cmd/aterm/climain.go

```diff
@@ -51,6 +51,9 @@ func main() {
 		return
 	}
 
+	// TOOD: replace version, owner, repo with real values
+	appdialogs.NotifyUpdate("v30.0.0", "google", "go-github")
```

Comment:
Code looks fine. Let's get #8 in and pull this data from the variables exposed by go-info and maybe setup a variable for the org/repo that we can populate from CI to automatically fill it in at build time like we do in the ashirt repo.